### PR TITLE
Add DerivingMeter to readme

### PR DESCRIPTION
**Pull request by malish8632**
Add missing DerivingMeter metric type to Readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -295,6 +295,32 @@ In addition to the tags that are specified (e.g., "what" and "endpoint" in this
 | unit        | \<unit\>/s |\<unit\> is what is originally specified as "unit" attribute during declaration. If missing, the value will be set as "n/s". For example if you originally specify .tagged("unit", "request") on a Meter, FfwdReporter emits Meter data points with "unit":"request/s"       |
 | stat | 1m, 5m    | **1m** means the size of the time bucket of the calculated moving average of this data point is 1 minute. **5m** means 5 minutes.         |
 
+
+## Deriving Meter
+It is a regular meter under the hood with slight difference.
```
Comment (lmuhlha) [inline README.md:300]: Just thought of a little rewording: 

A deriving meter takes the derivative of a value that is expected to be almost monotonically
increasing.
<BR>A typical use case is to get the rate of change of a counter of the total number of
events.
<BR>This implementation ignores updates that decrease the counter value. The rationale is that the counter is expected to be monotonically increasing between infrequent resets (when a process has been restarted, for example). Thus, negative values should only happen on restart and should be safe to discard.

Comment (lmuhlha) [reply README.md:300]: Was debating removing "almost" monotonically as well. Is the almost just referring to restarts?

File: README.md [hunk 2]
```diff
@@ -295,6 +295,32 @@ In addition to the tags that are specified (e.g., "what" and "endpoint" in this
...
+DerivingMeter derivingMeter = registry.derivingMeter(metric.tagged("what", "incoming-requests").tagged("endpoint", "/v1/list"));
+derivingMeter.mark();
+```
+
+In addition to the tags that are specified (e.g., "what" and "endpoint" in this example), FfwdReporter adds the following tags to each Meter data point:
+
+| tag         | values   | comment |
+|-------------|----------|---------|
+| metric_type | deriving_meter |         |
+| unit        | \<unit\>/s |\<unit\> is what is originally specified as "unit" attribute during declaration. If missing, the value will be set as "n/s". For example if you originally specify .tagged("unit", "request") on a DerivingMeter, FfwdReporter emits DerivingMeter data points with "unit":"request/s"       |
```
Comment (lmuhlha) [inline README.md:320]: \<unit\> is set to what is specified during declaration. For example, if you specify .tagged("unit", "request") on a DerivingMeter, FfwdReporter emits DerivingMeter data points with "unit":"request/s".  Default: "n/s".

File: README.md (hunk 1 above)
Comment (malish8632) [reply README.md:300]: I grabbed that description directly from the code, tbh. I can remove it if it hurts :)

File: README.md [hunk 3]
```diff
@@ -295,6 +295,32 @@ In addition to the tags that are specified (e.g., "what" and "endpoint" in this
...
+| stat | 1m, 5m    | **1m** means the size of the time bucket of the calculated moving average of this data point is 1 minute. **5m** means 5 minutes.         |
```
Comment (lmuhlha) [inline README.md:321]: \<stat\> is the size of the time bucket of the calculated moving average of this data point. **1m** is 1 minute. **5m** is 5 minutes.

Comment (malish8632) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (malish8632) [reply README.md]: fixed

File: README.md (hunk 1 above)
Comment (malish8632) [reply README.md]: removed and updated the desc

**Review by lmuhlha — APPROVED** [review]

**Pull request merged**